types/apexcharts.d.ts

```diff
@@ -40,9 +40,20 @@ declare class ApexCharts {
   addPointAnnotation(options: any, pushToMemory?: boolean, context?: any): void
   removeAnnotation(id: string, options?: any): void
   clearAnnotations(options?: any): void
-  dataURI(options?: { scale?: number, width?: number }): Promise<void>
+  dataURI(options?: { scale?: number, width?: number }): Promise<{ imgURI: string }>
   static exec(chartID: string, fn: string, ...args: Array<any>): any
+  static getChartByID(chartID: string): ApexCharts|undefined
   static initOnLoad(): void
+  exports: {
```

Comment:
How is this `exports` object relevant here? It is not publicly exposed in ApexCharts API. How are you using it?

Reply:
Even though it's not documented or anything, it's still exposed. In our case, we use it to perform diverse operations that are implemented but hardly accessible from userland.

We have a custom toolbar, which we want to expose SVG/PNG/CSV download buttons, among other things. So we use `instance.exports.exportToSVG()`, `instance.exports.triggerDownload()` and other methods directly - where `instance` is an ApexChart instance retrieved by `getChartByID`.